INSTALL.md

```diff
@@ -149,7 +130,7 @@ Every PR and every merge to master triggers the CI process in [Jenkins](http://j
 The Jenkins CI will build, run unit tests, run integration tests and Publish artifacts- On every commit to PR and master.
 If any of the CI stages fail, then the process is aborted and no artifacts are published.
 On every successful build Artifacts are pushed to a [s3 bucket](https://release.rook.io/). On every successful master build,
-images are uploaded to quay and docker hub in addition.  
+images are uploaded to quay and docker hub in addition.
```

Comment:
There is nothing on `quay` at the moment, we have the org, I'm just not sure if it's useful, @bassam, what do you think?

Reply:
you don't need to push to quay. However, rook is currently pushing to quay.